bindings/python/iota_sdk/types/essence.py

```diff
@@ -50,10 +51,18 @@ class RegularTransactionEssence:
     outputs: List[Output]
     context_inputs: Optional[List[ContextInput]] = None
     allotments: Optional[List[ManaAllotment]] = None
+    capabilities: HexStr = field(default='0x', init=False)
     payload: Optional[Payload] = None
     type: int = field(
         default_factory=lambda: EssenceType.RegularTransactionEssence,
         init=False)
 
+    def with_capabilities(self, capabilities: bytes):
+        """Sets the transaction capabilities from a byte array.
```

Comment:
There is also a `bytearray` type, comment or parameter should be aligned
`The difference between bytes() and bytearray() is that bytes() returns an object that cannot be modified, and bytearray() returns an object that can be modified.`

Reply:
True, nvm